subcommand/injector/flags_test.go

```diff
@@ -227,3 +227,31 @@ func TestCommandEnvInts(t *testing.T) {
 		})
 	}
 }
+
+func TestCommandEnvFloats(t *testing.T) {
+	var cmd Command
+	tests := []struct {
+		env    string
```

Comment:
nit: seems like this should be named `var` rather than env which usually associated with processes complete environment.

I see the previous tests follow this pattern. I think it is okay to make a break from it. That can be done in a separate PR, with all instances updated.